challenge_6.js

```diff
@@ -6,11 +6,20 @@
  */
 exports.SumOfASquare = SumOfASquare;
 
-function SumOfASquare(naturalNumbers){
+function SumOfASquare(naturalNumbers) {
   this.naturalNumbers = naturalNumbers;
-}
 
-// do work here
+  this.sumOfSquares = function() {
+    let sumNsquared = 0;
+    for (let i = 1; i <= this.naturalNumbers; i++) {
+      sumNsquared += i ** 2;
+    }
+    return sumNsquared;
+  };
 
-// overwrite the object prototype, or add methods to the prototype
-// to satisfy the two method calls, sumOfSquares() and squareOfTheSums()
+  this.squareOfTheSums = function() {
+    let squaredNsum =
```

Comment:
It's best to always comment above, the math equation you're using if it's not widely known to everyone: https://en.wikipedia.org/wiki/Triangular_number